packages/block-library/src/table/edit.js

```diff
@@ -418,23 +421,32 @@ export class TableEdit extends Component {
 
 		if ( isEmpty ) {
 			return (
-				<form onSubmit={ this.onCreateTable }>
-					<TextControl
-						type="number"
-						label={ __( 'Column Count' ) }
-						value={ initialColumnCount }
-						onChange={ this.onChangeInitialColumnCount }
-						min="1"
-					/>
-					<TextControl
-						type="number"
-						label={ __( 'Row Count' ) }
-						value={ initialRowCount }
-						onChange={ this.onChangeInitialRowCount }
-						min="1"
-					/>
-					<Button isPrimary type="submit">{ __( 'Create' ) }</Button>
-				</form>
+				<Placeholder
+					label={ __( 'Table' ) }
+					icon={ <BlockIcon icon={ icon } showColors /> }
+					instructions={ __( 'Insert a table for sharing data' ) }
```

Comment:
The instructions on our other placeholders (Image, Gallery, etc.) end with a period. Probably best that we stay consistent.

```suggestion
					instructions={ __( 'Insert a table for sharing data.' ) }
```